tribe-ext-calendar-widget-areas/index.php

```diff
@@ -144,26 +136,24 @@ public function get_enabled_areas_full_details() {
 		
 		$return = array();
 		
-		if ( is_array( $enabled_areas ) ) {
+		// if none selected, return all available
+		if ( empty( $enabled_areas ) ) {
+			$return = $all_available;
+		} else {
 			foreach ( $enabled_areas as $key => $value ) {
 				if ( array_key_exists( $value, $all_available ) ) {
 					$return[$value] = $all_available[$value];
 				}
 			}
 		}
 		
-		// if none selected, return all available
-		if ( empty( $enabled_areas ) || ! is_array( $enabled_areas ) ) {
-			$return = $all_available;
-		}
-		
 		return $return;
 	}
 	
 	/**
 	 * Add options to Tribe settings page
 	 *
-	 * @access: public
+	 * @see Tribe__Extension__Settings_Helper
```

Comment:
We probably don't need an @see for this. I have never seen us use an @see unless it applies to the function args or the return, except in rare cases.